src/Features/ShippingLabelBanner.php

```diff
@@ -44,33 +39,36 @@ public function __construct() {
 	 * @return bool
 	 */
 	private function should_show_meta_box() {
-		$order = wc_get_order();
+		if ( ! $this->shipping_label_banner_display_rules ) {
+			$jetpack_version   = null;
+			$jetpack_connected = null;
+			$wcs_version       = null;
+			$wcs_tos_accepted  = null;
 
-		if ( ! $order ) {
-			return false;
-		}
-		// Restrict showing the metabox to supported store currencies.
-		$base_currency = get_woocommerce_currency();
-		if ( ! $this->is_supported_currency( $base_currency ) ) {
-			return false;
-		}
+			if ( class_exists( 'Jetpack_Data' ) ) {
+				$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
 
-		$base_location = wc_get_base_location();
-		if ( ! $this->is_supported_country( $base_location['country'] ) ) {
-			return false;
-		}
+				$jetpack_connected = isset( $user_token->external_user_id );
+				$jetpack_version   = JETPACK_VERSION;
```

Comment:
I'm seeing this defined as `JETPACK__VERSION` (double underscore) not `JETPACK_VERSION`